libraries/image/src/custom-mobilenet.ts

```diff
@@ -314,14 +304,29 @@ export async function loadTruncatedMobileNet(modelOptions?: ModelOptions) {
     }
 }
 
-export async function load(checkpoint: string, metadata?: string | Metadata ) {
-    const customModel = await tf.loadLayersModel(checkpoint);
-    const metadataJSON = metadata ? await processMetadata(metadata) : null;
+export async function load(model: string, metadata?: string | Metadata) {
```

Comment:
load now supports adding a global URL from TM2 OR the individual model/metadata urls.